framework/commercetools/api/endpoints/logout/index.ts

```diff
@@ -0,0 +1,18 @@
+import { GetAPISchema, createEndpoint } from '@commerce/api'
+import logoutEndpoint from '@commerce/api/endpoints/logout'
+import type { LogoutSchema } from '../../../types/logout'
```

Comment:
Is possible to add alias to avoid ../../

Reply:
Done